lib/rules/indentation/html.test.js

```diff
@@ -256,86 +276,95 @@ testRule({
 
 	accept: [
 		{
-			code: `
-<style>
-\ta {
-\t\tdisplay:block;
-\t}
-</style>`,
-		},
-		{
-			code: `
-\t<style>
-\t\ta {
-\t\t\tdisplay:block;
-\t\t}
-\t</style>`,
-		},
-		{
-			code: `
-<style lang="less" nonce="1">
-\ta {
-\t\tdisplay:block;
-\t}
-</style>`,
-		},
-		{
-			code: `
-<style
-\tlang="less"
-\tnonce="1">
-\ta {
-\t\tdisplay:block;
-\t}
-</style>`,
-		},
-		{
-			code: `
-<style
-\t\tlang="less"
-\t\tnonce="1"
->
-\ta {
-\t\tdisplay:block;
-\t}
-</style>`,
-		},
-		{
-			code: `
-\t<style
-\t\tlang="less"
-\t\tnonce="1"
-\t>
-\t\ta {
-\t\t\tdisplay:block;
-\t\t}
-</style>`,
-		},
-		{
-			code: `
-<style
-\tlang="less"
-\t\tnonce="1">
-\ta {
-\t\tdisplay:block;
-\t}
-</style>`,
+			code: stripIndent(`
+				<style>
+					a {
+						display:block;
+					}
+				</style>
+			`),
+		},
+		{
```

Comment:
Same as my previous comment - if the test checks leading tabs/spaces, it's better to leave it as it is, since `stripIndent` cuts such leading indentation, and it affects the logic